src/main/Config.cpp

```diff
@@ -142,6 +142,7 @@ Config::Config() : NODE_SEED(SecretKey::random())
     EXPERIMENTAL_BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT = 14; // 2^14 == 16 kb
     EXPERIMENTAL_BUCKETLIST_DB_INDEX_CUTOFF = 20;             // 20 mb
     EXPERIMENTAL_BUCKETLIST_DB_PERSIST_INDEX = true;
+    PUBLISH_TO_ARCHIVE_DELAY = std::chrono::seconds{2};
```

Comment:
actually, should we set this to 0 by default, and operators that need this functionality can adjust the config manually?